packages/block-library/src/heading/block.json

```
@@ -35,7 +35,15 @@
 		"typography": {
 			"fontSize": true,
 			"lineHeight": true,
-			"__experimentalFontWeight": true
+			"__experimentalFontStyle": true,
+			"__experimentalFontWeight": true,
+			"__experimentalLetterSpacing": true,
+			"__experimentalTextTransform": true,
+			"__experimentalDefaultControls": {
+				"fontSize": true,
+				"fontAppearance": true,
+				"__experimentalTextTransform": true
```

There aren't nested `__experimental` prefixes on `__experimentalDefaultControls`.  As it currently sits, the text-transform control does not show by default.

```suggestion
				"textTransform": true
```